tests/python/gaia-ui-tests/gaiatest/apps/settings/regions/device_info.py

```diff
@@ -42,12 +70,58 @@ def model(self):
     def software(self):
         return self.marionette.find_element(*self._software_locator).text
 
+    def tap_your_rights(self):
+        Wait(self.marionette).until(expected.element_displayed(
+            self.marionette.find_element(*self._your_rights_locator)))
```

Comment:
You're using self.marionette.find_element(*self._your_rights_locator) 2 times.
So you should use 
element = self.marionette.find_element(*self._your_rights_locator)
Wait(self.marionette).until(expected.element_displayed(element))
element.tap()